pkg/controllers/failuredomainrollout/flags.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2025 Nutanix. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package failuredomainrollout
+
+import (
+	"github.com/spf13/pflag"
+)
+
+type Options struct {
+	Enabled     bool
+	Concurrency int
+}
+
+func (o *Options) AddFlags(flags *pflag.FlagSet) {
```

Comment:
Please add some Helm chart values for this, similar to 

https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/019937438be098b1a054d4c13ea59169bb0fa1ad/charts/cluster-api-runtime-extensions-nutanix/values.yaml#L129-L137